.github/workflows/release.yml

```diff
@@ -0,0 +1,105 @@
+name: Releases
+
+on:
```

Comment:
I'd like this to also run on PRs, but without uploading the artifacts to the release.